packages/components/TopMenu/NotificationDrawer.tsx

```diff
@@ -0,0 +1,370 @@
+  View,
+  ViewStyle,
+} from "react-native";
+import { BellAlertIcon, BellIcon } from "react-native-heroicons/outline";
+import { XMarkIcon } from "react-native-heroicons/solid";
+
+import { TOP_MENU_BUTTON_HEIGHT } from "./TopMenu";
+import {
+  DismissNotificationRequest,
+  Notification,
+  NotificationsRequest,
+} from "../../api/notification/v1/notification";
+import { useDropdowns } from "../../context/DropdownsProvider";
+import { useNSUserInfo } from "../../hooks/useNSUserInfo";
+import {
+  notificationsQueryKey,
+  useNotifications,
+} from "../../hooks/useNotifications";
+import useSelectedWallet from "../../hooks/useSelectedWallet";
+import { parseUserId } from "../../networks";
+import { mustGetNotificationClient } from "../../utils/backend";
+import {
+  neutral00,
+  neutral33,
+  neutral44,
+  primaryColor,
+  secondaryColor,
+} from "../../utils/style/colors";
+import {
+  fontSemibold12,
+  fontSemibold14,
+  fontSemibold8,
+} from "../../utils/style/fonts";
+import { headerHeight, layout, topMenuWidth } from "../../utils/style/layout";
+import { tinyAddress } from "../../utils/text";
+import { BrandText } from "../BrandText";
+import { OmniLink } from "../OmniLink";
+import { UserNameInline } from "../UserNameInline";
+import { TertiaryBox } from "../boxes/TertiaryBox";
+
+export const NotificationDrawer: React.FC = () => {
+  const { onPressDropdownButton, isDropdownOpen } = useDropdowns();
+
+  const dropdownRef = useRef<View>(null);
+  const selectedWallet = useSelectedWallet();
+  const notifications = useNotifications({
+    userId: selectedWallet?.userId,
+  });
+
+  return (
+    <View ref={dropdownRef}>
+      <TouchableOpacity onPress={() => onPressDropdownButton(dropdownRef)}>
+        <TertiaryBox
+          width={50}
+          style={{
+            marginRight: layout.spacing_x1_5,
+          }}
+          mainContainerStyle={{
+            justifyContent: "space-between",
+            flexDirection: "row",
+            paddingHorizontal: 12,
+            backgroundColor: isDropdownOpen(dropdownRef)
+              ? neutral33
+              : neutral00,
+          }}
+          height={TOP_MENU_BUTTON_HEIGHT}
+        >
+          {notifications && notifications.length > 0 ? (
+            <>
+              <View
+                style={{
+                  position: "absolute",
+                  top: 6,
+                  right: 4,
+                  borderRadius: 9999,
+                  width: 8,
+                  height: 8,
+                  backgroundColor: primaryColor,
+                }}
+              >
+                <BrandText
+                  style={[
+                    fontSemibold8,
+                    {
+                      position: "absolute",
+                      right: 2.3,
+                    },
+                  ]}
+                >
+                  {notifications.length > 9 ? "♾️" : notifications.length}
+                </BrandText>
+              </View>
```

Comment:
I purpose something like this

```suggestion
                            <View
                style={{
                  position: "absolute",
                  top: -2,
                  right: -2,
                  alignItems: "center",
                  justifyContent: "center",
                  borderRadius: 9999,
                  height: 14,
                  paddingHorizontal: layout.spacing_x0_75,
                  backgroundColor: primaryColor,
                }}
              >
                <BrandText
                  style={[
                    fontSemibold10,
                    {
                    },
                  ]}
                >
                  {notifications.length > 99 ? "+99" : notifications.length}
                </BrandText>
              </View>
```

Reply:
can you provide a screenshot side by side? thanks in advance